test/language/generators/syntax/yield-as-binding-identifier.js

```diff
@@ -0,0 +1,14 @@
+// Copyright (C) 2013 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+  description: >
+      `yield` is a reserved keyword within generator function bodies and may
+      not be used as a binding identifier.
```

Comment:
This description is not correct. In this case `yield` is an `IdentitfierExpression`.